examples/export-in-node/src/index.ts

```diff
@@ -1,10 +1,13 @@
 import * as dotenv from "dotenv";
 import * as path from "path";
-import { input } from "@inquirer/prompts";
+import prompts from "prompts";
 import { Turnkey } from "@turnkey/sdk-server";
 import { Crypto } from "@peculiar/webcrypto";
 import { generateP256KeyPair, decryptExportBundle } from "@turnkey/crypto";
-global.crypto = new Crypto();
+if (typeof crypto === "undefined") {
```

Comment:
@andrewkmin @moe-dev Tagging for context

This was the fix for the following error that I ran into while running this example: 

`TypeError: Cannot set property crypto of #<Object> which has only a getter`

Reply:
Same for `import-in-node`

Reply:
Node version --> `v22.5.1`

Pnpm version --> `8.4.0`

Reply:
Posting quick follow up for visibility --> We tested this issue and it seems like this is a node versioning issue 

Old code works with node 18 but not with node 20